sklearn/metrics/tests/test_classification.py

```diff
@@ -2470,6 +2470,7 @@ def test_log_loss():
         [[0.5, 0.5], [0.1, 0.9], [0.01, 0.99], [0.9, 0.1], [0.75, 0.25], [0.001, 0.999]]
     )
     loss = log_loss(y_true, y_pred)
+    # -np.mean(bernoulli.logpmf(np.array(y_true) == "yes", y_pred[:, 1]))
```

Comment:
leftover

Reply:
No leftover. It is a comment on how to arrive at the magic umber 1.8817971.

Reply:
Oh I see. Would it make sense to compute this expression here, instead of hard coding the result?
Otherwise I would write more explicitly `# 1.8817971 = -np.mean(bernoulli.logpmf(np.array(y_true) == "yes", y_pred[:, 1]))`.